packages/adapter-nuxt/src/hooks/snippet-read.ts

```diff
@@ -58,19 +58,38 @@ export const snippetRead: SnippetReadHook<PluginOptions> = async (
 		}
 
 		case "Link": {
+			const repeat = data.model.config?.repeat ?? false;
+			const allowText = data.model.config?.allowText ?? false;
+
+			let codeText;
+			if (!repeat && !allowText) {
+				codeText = stripIndent`
+					<PrismicLink :field="${dotPath(fieldPath)}">Link</PrismicLink>
+				`;
+			} else if (!repeat && allowText) {
+				codeText = stripIndent`
+					<PrismicLink :field="${dotPath(fieldPath)}" />
+				`;
+			} else if (repeat && !allowText) {
+				codeText = stripIndent`
+					<template v-for="(link, index) in ${dotPath(fieldPath)}" :key="index">
+						<PrismicLink :field="link">Link</PrismicLink>
+					</template>
+				`;
+			} else if (repeat && allowText) {
+				codeText = stripIndent`
+					<template v-for="(link, index) in ${dotPath(fieldPath)}" :key="index">
+						<PrismicLink :field="link" />
+					</template>
+				`;
```

Comment:
Not sure why, but I wasn't able to see the correct snippet with repeat, does it work for you?

![Screenshot 2024-11-12 at 10 11 35](https://github.com/user-attachments/assets/597f0b7a-6b48-4f5d-a4e8-d5c613ccc3f8)